core/cap-0058.md

```diff
@@ -277,11 +277,11 @@ Contract re-initialization is risky for a lot of cases, e.g. for contracts with
 
 While in some cases additional initialization may be required after Wasm has been updated, we deem the risks of allowing it to be higher than the benefits for such cases. Unlike initialization, atomicity is not as important, because the authorization priviliges don't change between the call to update the contract and the call to re-initialize it. Thus a programmatic, contract-specific solution should be feasible.
 
-### `SOROBAN_AUTHORIZED_FUNCTION_TYPE_CREATE_CONTRACT_HOST_FN` deprecation
+### `SOROBAN_AUTHORIZED_FUNCTION_TYPE_CREATE_CONTRACT_HOST_FN` is no longer supported
```

Comment:
I started a new discussion thread on this topic:
- https://github.com/stellar/stellar-protocol/discussions/1501#discussioncomment-10423791